pygromos/files/topology/mtb.py

```diff
@@ -5,24 +5,29 @@
 Author: Marc Lehner
 """
 
-from typing import Dict, List
-import warnings
+from typing import List
 from pygromos.files._basics import _general_gromos_file
 from pygromos.files.blocks import mtb_blocks as blocks
 
 
 class Mtb(_general_gromos_file._general_gromos_file):
     _gromos_file_ending: str = "mtb"
     MTBUILDBLSOLUTE_list: List[blocks.MTBUILDBLSOLUTE]
+    MTBUILDBLSOLVENT_list: List[blocks.MTBUILDBLSOLVENT]
+    MTBUILDBLEND_list: List[blocks.MTBUILDBLEND]
 
     def __init__(self, in_value: (str or dict or None), _future_file: bool = False):
         self.MTBUILDBLSOLUTE_list = []
+        self.MTBUILDBLSOLVENT_list = []
+        self.MTBUILDBLEND_list = []
```

Comment:
dicts over lists here?

Reply:
But what would the dict keys be?
- Residue name
- Molecule name
- Position in MTB
- ....

Reply:
blockname? - like residue name

Reply:
so this would translate to e.g.: mtb_file.MTBUILDBLSOLUTE_list['ALA']

Reply:
but besides this detail, I love the implementation! :)

Reply:
one could store the dict as mtb_file.MTBUILDBLSOLUTE (and keep the list if desired)